Amazon.IonObjectMapper.Demo/CaseInsensitiveBug.cs

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+namespace Amazon.IonObjectMapper.Demo
+{
+    using System;
+    using System.IO;
+    using Amazon.IonObjectMapper.Test;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+    public class PluralCaseClass 
```

Comment:
I don't think we should add this test since the assumption in ObjectMapper is that the user will following the naming convention in C# to use pascal casing for public properties and fields https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#pascal-case. And we've already demoed `PropertyNameCaseInsensitive` option in `Amazon.IonObjectMapper.Demo/PropertyNameCaseInsensitiveTest.cs`

Reply:
Removed `CaseInsensitiveBug`